apps/in_hand_scanner/src/integration.cpp

```diff
@@ -372,6 +368,12 @@ pcl::ihs::Integration::merge (const CloudXYZRGBNormalConstPtr& cloud_data,
       // 4 - 2   1  //
       //   \   \    //
       // *   3 - 0  //
+
+      VertexIndex& vi_1 = vertex_indices [ind_1];
```

Comment:
@taketwo Is there a reason why `vi_0` is conditionally re-assigned and others aren't? I've not seen the algo, but based on the diagram in the comment, I'd expect there to be changes in other vertex indices based on index used for `vi_0`

Reply:
This does look strange, but without knowing the algorithm it's impossible to say whether it's wrong :man_shrugging: